client/layout/masterbar/notifications.jsx

```diff
@@ -117,6 +121,7 @@ const MasterbarItemNotifications = React.createClass( {
 				icon="bell"
 				onClick={ this.toggleNotesFrame }
 				isActive={ this.props.isActive }
+				isNotesItem={ true }
```

Comment:
Just thinking out loud: could something like `renderAsAnchor` be a better name for this `isNotesItem` prop?